_posts/2017-09-15-eslint-v4.7.0-released.md

```diff
@@ -7,11 +7,15 @@ tags:
 ---
 # ESLint v4.7.0 released
 
-We just pushed ESLint v4.7.0, which is a minor release upgrade of ESLint. This release adds some new features andfixes several bugs found in the previous release. 
-
+We just pushed ESLint v4.7.0, which is a minor release upgrade of ESLint. This release adds some new features and fixes several bugs found in the previous release.
```

Comment:
The ejs blogpost template doesn't add a space in "andfixes" here, but I don't know ejs well enough for it to be a ten-second fix.